src/20_Components/amp-social-share.html

```diff
@@ -32,37 +40,53 @@
+    <amp-social-share type="email"></amp-social-share>
+    <amp-social-share type="pinterest"></amp-social-share>
+    <amp-social-share type="linkedin"></amp-social-share>
+    <amp-social-share type="facebook" data-param-app_id="254325784911610"></amp-social-share>
   </div>
 
   <!-- #### Configuration -->
   <!--
     Embed the `amp-social-share` widget [choosing a type](https://www.ampproject.org/docs/reference/components/amp-social-share#types), then configure the actions.
 
-    * `text` is the text to include in the share
-    * `url` is the URL to share
-    * `attribution` is where the share is attributed to.
+    * `width`, default 60px.
+    * `height`, default 44px.
+    * `data-param-text` is the text to include in the share.
+    * `data-param-url` is the URL to share.
+    * `data-param-attribution` is where the share is attributed to.
+
+    All `data-param-*` prefixed attributes will be turned into URL parameters and passed to the share endpoint.
   -->
-  <amp-social-share type="linkedin" width="60" height="44"
-    data-text="Check out these AMP Examples!"
-    data-url="https://ampbyexample.com/"
-    data-attribution="AMPhtml">
+  <amp-social-share type="linkedin"
+    width="40"
+    height="40"
+    data-param-text="Check out these AMP Examples!"
+    data-param-url="https://ampbyexample.com/"
+    data-param-attribution="AMPhtml">
   </amp-social-share>
 
-  <!-- #### Customized Views -->
+  <!-- #### Custom Style -->
   <!--
-    Sometimes you want to provide your own style. In this instance, you can **embed an anchor without a `href`** and it will be populated. This provides you the flexibility to build your own UI for the share button.
+    Whenever you want to provide your own style, use CSS properties within an `<style amp-custom>` ([Modify the presentation](https://www.ampproject.org/docs/get_started/create/presentation_layout)) element. This will make sure the image stays responsive and centered, and the default styling is overwritten.
+
+    * `background-color`, if you want the element to be a different color.
+    * `background-image`, provide another image if you want to change the icon.
+    * `background-repeat` (when using `background-image`), set to `no-repeat`.
+    * `background-position` (when using `background-image`), set to `center`.
+    * `background-size` (when using `background-image`), set to `contain`.
   -->
-  <amp-social-share type="linkedin" width="60" height="44" data-text="The AMP Project" data-url="https://www.ampproject.org/" data-attribution="amphtml">
-    <a id="customized-social-share" class="custom">
-      <amp-img src="https://raw.githubusercontent.com/google/material-design-icons/master/social/1x_web/ic_share_white_48dp.png" width="18" height="18" layout="responsive" alt="an image">
-      </amp-img>
-    </a>
+  <amp-social-share type="linkedin"
+    width="76"
+    height="22"
+    class="custom-style">
+  </amp-social-share>
+  <amp-social-share type="linkedin"
```

Comment:
Not really, wasn't sure I was gonna commit the two examples. I was thinking it would make it more clear to the viewer that it's responsive and works the same as the "original" styling, but I don't think it's our job to teach people common CSS. Removed the second example, it's implied that it works the same as the out-of-the-box styling.